contrib/linux/sensors/tail/tail.py

```diff
@@ -0,0 +1,325 @@
+# Copyright 2014 Koert van der Veer
```

Comment:
Because of that, I assume this failed is based on logshipper?

Reply:
Also, how hard would it be / would it make sense to contribute those changes upstream?

Reply:
Yes, I copied and pasted that file out of logshipper. Both StackStorm and logshipper are licensed under the Apache 2.0 license though, so there shouldn't be a legal issue.

Upstream logshipper hasn't been updated in 3-4 years. The author hasn't responded to any issues or pull requests in that long, and the last commit was on Mar 23, 2015. I would be happy to upstream this if I thought it had a chance to get merged, but I don't think the original author is interested in the project anymore.

Reply:
If possible, I would prefer to split out / re-organize the code so it's easy to see which code is original and which is our modifications.

I know that might not be possible though, or at least not easily.